app/cache/cache-config.js

```diff
@@ -0,0 +1,8 @@
+const CacheService = require('./cache-service');
```

Comment:
I'd rename this file to `user-info-cache`. 
`const userInfoCache = require('../cache/cache-config').userInfoCache;` 
doesn't read well

Reply:
The cache config can be used for other caches in future too (a central file for cache configurations), but will change to es6 import to make readability better 👍